### DISCUSSION TITLE
Simulating periodic domain

### DISCUSSION THREAD

**Discussion by mishkat1096** [neperfepx/FEPX#17] [Q&A] [answered]
Hello,
I am trying to use FEPX to simulate a periodic domain generated by neper tesselation.
However, when I run the simulation I run into an error saying `Surface allocation error.`
I am using the input file from the uniaxial example.

Thanks

**Comment by mkasemer** [comment, marked as answer]
Hello @mishkat1096,

Currently FEPX does not support periodic tessellations. This is a feature which is currently being developed, but for now you must use tessellations within rectangular prismatic domains.

**Reply by mishkat1096** [reply]
Okay. Thanks
